package.json

```diff
@@ -40,6 +40,7 @@
     "react-dom": "^18.2.0",
     "react-ga4": "^2.1.0",
     "react-markdown": "^5.0.3",
+    "react-pdf": "5.x",
```

Comment:
This is the latest version of this package that I could get to work with our current setup.